big-practice/src/js/pages/dashboard.page.js

```diff
@@ -1,114 +1,115 @@
-const modalForm = document.querySelector('.modal-form');
-const modalWarning = document.querySelector('.modal-warning');
-const layoutMainAddBtn = document.querySelector('.layout-main .add-btn');
-const modalFormCreateBtn = document.querySelector('.modal-form .create-btn');
-const modalFormUpdateBtn = document.querySelector('.modal-form .update-btn');
-const modalFormCancelBtn = document.querySelector('.modal-form .cancel-btn');
-const modalWarningCancelBtn = document.querySelector('.modal-warning .cancel-btn');
-const modalWarningDeleteBtn = document.querySelector('.modal-warning .delete-btn');
-const logoutBtn = document.querySelector('.site-header .logout-btn');
-const loginPage = 'login.html';
-
-/**
+import { DocumentHelper } from '../helpers/document.helper';
+import { USERNAME_KEY, LOGIN_PAGE } from '../constants/app.constant';
+
+class Dashboard {
+  // Query elements
+  tableBody = document.getElementById('tableBody');
+  nameMovieInput = document.getElementById('nameMovie');
+  directorInput = document.getElementById('director');
+  nationInput = document.getElementById('nation');
+  accountName = document.querySelector('.account-name');
+  form = document.querySelector('.form');
+  modalForm = document.querySelector('.modal-form');
+  modalWarning = document.querySelector('.modal-warning');
+  layoutMainAddBtn = document.querySelector('.layout-main .add-btn');
+  modalFormCreateBtn = document.querySelector('.modal-form .create-btn');
+  modalFormUpdateBtn = document.querySelector('.modal-form .update-btn');
+  modalFormCancelBtn = document.querySelector('.modal-form .cancel-btn');
+  modalWarningCancelBtn = document.querySelector('.modal-warning .cancel-btn');
+  modalWarningDeleteBtn = document.querySelector('.modal-warning .delete-btn');
+  logoutBtn = document.querySelector('.site-header .logout-btn');
+
+  /**
  * Hide modal
  *
  * @param {element} element - Element of modal
  */
-const hideModal = (element) => {
-  element.classList.remove('modal-show');
-};
-
-/**
- * Show modal
- *
- * @param {element} element - Element of modal
- */
-const showModal = (element) => {
-  element.classList.add('modal-show');
-};
-
-/**
- * Hide element
- *
- * @param {element} element - Element
- */
-const hideElement = (element) => {
-  element.classList.add('hide');
-};
+  hideModal(element) {
+    element.classList.remove('modal-show');
+  }
 
-/**
- * Show element
- *
- * @param {element} element - Element
- */
-const showElement = (element) => {
-  element.classList.remove('hide');
-};
+  /**
+   * Show modal
+   *
+   * @param {element} element - Element of modal
+   */
+  showModal(element) {
+    element.classList.add('modal-show');
+  }
 
-/**
- * Clean error message
- *
- * @param {HtmlInputElement} element - Element input
- */
-const cleanErrorMessage = (element) => {
-  showErrorMessage(element, EMPTY_TEXT);
-};
+  /**
+   * Hide element
+   *
+   * @param {element} element - Element
+   */
+  hideElement(element) {
```

Comment:
Create new helper for showing/hiding element

Reply:
Updated!